### PR TITLE
List all load balancers

### PR DESCRIPTION
**Pull request by palanik1**
This PR adds functionality LBList() for listing all Load Balancers 

Co-authored by : Sayandeep Sen sayandes@in.ibm.com

Signed-off-by: Palani Kodeswaran <palankod@in.ibm.com>

### CODE REVIEW

File: load_balancer.go [hunk 1]
```diff
@@ -159,6 +159,28 @@ func (odbi *ovndb) lbGetImp(name string) ([]*LoadBalancer, error) {
...
+
+	cacheLoadBalancer, ok := odbi.cache[TableLoadBalancer]
+	if !ok {
+		return nil, ErrorSchema
+	}
+
+	listLB := make([]*LoadBalancer, 0, len(cacheLoadBalancer))
+	for uuid := range cacheLoadBalancer {
+		lb ,err := odbi.rowToLB(uuid)
+		if(err != nil){
```
Comment (vtolstov) [inline load_balancer.go:174]: needs gofmt, please gofmt all files

**Review by vtolstov — CHANGES_REQUESTED** [review]
after gofmt will be merged

File: load_balancer.go (hunk 1 above)
Comment (palanik1) [reply load_balancer.go]: Done.

**Comment by palanik1** [comment]
@vtolstov Addressed your comments. Please take a look. Thanks.

**Review by vtolstov — APPROVED** [review]

**Comment by vtolstov** [comment]
merged as it simple change

**Comment by palanik1** [comment]
@vtolstov Thank you for your reviews and quick response.

**Pull request merged**